### ISSUE TITLE
docker-compose fails if runc is not available in environment.

### ISSUE THREAD

**Issue by jthomas** [apache/incubator-openwhisk-devtools#81]
## issue 
devtools [assumes runc is available](https://github.com/apache/incubator-openwhisk-devtools/blob/master/docker-compose/Makefile#L23-L29) in the environment and references the expected path when [creating volumes](https://github.com/apache/incubator-openwhisk-devtools/blob/master/docker-compose/docker-compose.yml#L119) for the invoker. 

I'm running the project on a Mac with Docker but without runc.

In the `local.env` file generated, `RUNC_BINARY` is empty.

```
RUNC_BINARY=
```

Running `make quick-start` on OS X then fails when the invoker can't be started. 

```
$ make quick-start
...
Creating openwhisk_invoker_1         ... error
ERROR: for openwhisk_invoker_1  Cannot create container for service invoker: create .: volume name is too short, names should be at least two alphanumeric characters
```

## workaround

Commenting out the volume definition, referencing the RUNC_BINARY variable, allows the invoker to start. Since RUNC support defaults to off, this does not affect the system.

```
     volumes:
       - ~/tmp/openwhisk/invoker/logs:/logs
       - /var/run/docker.sock:/var/run/docker.sock
       - /var/lib/docker/containers:/containers
#      - ${RUNC_BINARY}:/usr/bin/docker-runc
#       - /run/runc:/run/runc
       - /sys/fs/cgroup:/sys/fs/cgroup
```

## solutions

- As runc support is an optional setting and not enabled in the Makefile, should this volume be removed by default? We could include instructions in the README for how to enable it.

Will submit a PR if others agree or hear about better solutions....

**Comment by jthomas** [comment]
@ddragosd can I get a review on this?

**Comment by ddragosd** [comment]
we merged #84 that should fix this. The latest docker release changed the location of `runc` and it can't be mounted OOTB, so removing the docker volume for now unblocks this, but it will introduce back the issue where action containers can't be reused.
